rpm_spec/subpackages/manageiq-core-services

```diff
@@ -0,0 +1,23 @@
+%package core-services
+Summary: %{product_summary} Core Services
+
+Requires: %{name}-core = %{version}-%{release}
+
+%description core-services
+%{product_summary} Core Services
+
+%post core-services
+# Remove any old systemd unit files
```

Comment:
Can we make this a little clearer - Maybe this?

```suggestion
# Remove any old dynamic systemd unit files prior to the switch to static systemd unit files - See  ManageIQ/manageiq#20983
```

Reply:
:+1: done